airbyte-integrations/connectors/source-salesforce/unit_tests/api_test.py

```diff
@@ -46,7 +47,7 @@
 
 _A_CHUNKED_RESPONSE = [b"first chunk", b"second chunk"]
 _A_JSON_RESPONSE = {"id": "any id"}
-_A_SUCCESSFUL_JOB_CREATION_RESPONSE = {"state": "JobComplete"}
+_A_SUCCESSFUL_JOB_CREATION_RESPONSE = SalesforceJobResponseBuilder().with_id("any id").with_state("JobComplete").get_response()
```

Comment:
nit: if the ID is not important, we shouldn't have the `with_id` here and the builder should put any value. The same comment applies at a couple of other places like [here](https://github.com/airbytehq/airbyte/pull/37340/files#diff-f796cd7457a59372f87a40a682234d81223682782816ee84101cd8c7530cee45R186)

Reply:
done!

Reply:
fixed